[GH#45384] Cleans up documentation mistakes on uninstalling GitOps. (#48774)
openshift/openshift-docs: modules/go-deleting-argocd-instance.adoc

[source,terminal]
----
$ oc delete gitopsservice cluster
----

[NOTE]
====
You cannot delete an Argo CD cluster from the web console UI.
====

> After the above command runs successfully the default GitOps Service and its respective Argo CD instance will be deleted from the `openshift-gitops` namespace.

Reviewer: We don't use "above" to indicate a relative location in our documents (see https://www.ibm.com/docs/en/ibm-style?topic=word-usage#above if you're interested). Instead, maybe revise to the following: 

After the previously command runs successfully, the default GitOps Service and its respective Argo CD instance will be deleted from the `openshift-gitops` namespace.

> After the previous command runs successfully the default GitOps Service and its respective Argo CD instance will be deleted from the `openshift-gitops` namespace.

Reviewer 2: Maybe we should just say `gitopsservice` instead of `GitOps Service` here on the off chance someone confuses it with the actual [GitOps Service on the appstudio side](https://github.com/redhat-appstudio/managed-gitops#managed-gitops-service-monorepo)

Author: Done.

> After the previous command runs successfully the default `gitopsservice` and its respective Argo CD instance will be deleted from the `openshift-gitops` namespace.

Reviewer 3: ```suggestion
After the previous command runs successfully, the  `gitopsservice` default service and its respective Argo CD instance is deleted from the `openshift-gitops` namespace.
```

> Delete any other custom ArgoCD instances from other namespaces using the following command:

Reviewer: Slight revision: 

Delete any other custom ArgoCD instances from other namespaces by entering the following command: 

All commands should be introduced with "by entering" or "by running" (see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output if you're interested).
